File: dynamic_listings/python/default/collisiondetection/point_rectangle.py
def point_rect_collision(x1: float, y1: float, rect_x: float, rect_y: float, rect_width: float, rect_height: float):
    # We check if the point is inside the rectangle
    return rect_x <= x1 <= rect_x + rect_width and rect_y <= y1 <= rect_y + rect_height

File: dynamic_listings/rust/default/collisiondetection/point_rectangle.rs
# #![allow(dead_code)]
fn point_rect_collision(
    x1: f32,
    y1: f32,
    rectx: f32,
    recty: f32,
    rectwidth: f32,
    rectheight: f32,
) -> bool {
    // We check if the point is inside the rectangle
    x1 >= rectx && x1 <= rectx + rectwidth && y1 >= recty && y1 <= recty + rectheight
}
